### PR TITLE
Dockerized setup of yearn-exporter + prometheus + grafana

### PR DESCRIPTION
**Pull request by mariuspod**
Hey,
I've merged some code from:
https://github.com/DarkGhost7/yearn-exporter
and the dashboards from:
https://github.com/DarkGhost7/yearn-exporter-info/tree/master/Dashboards

and added a Dockerfile + docker-compose.yml so it can be run as part of a CI/CD pipeline.
Updated README with instructions how to run everything locally.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    networks:
+      - yearn-exporter
+    restart: always
+
+  prometheus:
+    image: prom/prometheus:v2.23.0
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+      - prometheus_data:/prometheus
+    command:
```
Comment (banteg) [inline docker-compose.yml:71]: Prometheus deletes data after two weeks by default, add `--storage.tsdb.retention.time 1y`

File: yearn/vaults_v2.py [hunk 2]
```diff
@@ -42,6 +46,7 @@
     "totalDebt",
 ]
 
+REGISTRY_DEPLOYMENT_BLOCK = 11563389
```
Comment (banteg) [inline yearn/vaults_v2.py:49]: I'll add proper v2 registry support in a separate pull request.

**Review by banteg — APPROVED** [review]
Looks good, added a few notes.

File: docker-compose.yml (hunk 1 above)
Comment (mariuspod) [reply docker-compose.yml:71]: haha that's a good catch, this would have been funny after two weeks :facepalm:

**Pull request merged**